### PR TITLE
CED-1422 Update EsFileThumbnail styling

### PR DESCRIPTION
**Pull request by lgeggleston**
### 🔗 Linked issue

https://energysage.atlassian.net/browse/CED-1422

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Style updates on EsFileThumbnail according to the Figma
- colors on spinner and progressCircle, text size, make preview link around the file name instead of separate
- Moved almost all styles to a separate scss file

### 🥼 Testing

Locally at http://localhost:8500/organisms/es-file-thumbnail
- On all viewports; and also try messing with the loading state example to see all states

#### 🧐 Feedback Requested / Focus Areas

- 

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.
- [x] I have documented testing approach

### CODE REVIEW

**Review by ericdouglaspratt — CHANGES_REQUESTED** [review]
I never realized how weird this component is - like the hardcoded bit of HTML just to generate a mobile preview. But fixing all that is definitely out of scope so nice work on the tweaks!

Just a couple small items.

**Review by jaissica — COMMENTED** [review]
The changes look good and matches the design. Two differences which I noticed were:
1. For thumbnails for file uploads the height and width currently is 156 * 156, but the figma suggests it to be 130 * 130.
2. The progress percentage inside the progress circle currently has - font-size: 16 px, font-weight: 800 and color: gray-700, but as per the figma it suggests for font-size: 12px, font-weight: 600 & color: black.

File: es-bs-base/scss/_file-thumbnail.scss [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  }
+}
+
+.thumbnail-spinner {
+  width: 40px;
+  height: 40px;
+  border-width: 0.3rem;
+}
+
+.icon-button {
```
[the remark this answers is not in the document]
Comment (lgeggleston) [reply es-bs-base/scss/_file-thumbnail.scss]: Ah yep, good idea

**Comment by lgeggleston** [comment]
> The progress percentage inside the progress circle currently has - font-size: 16 px, font-weight: 800 and color: gray-700, but as per the figma it suggests for font-size: 12px, font-weight: 600 & color: black (this is related to ESProgessCircle not sure if we would do in this ticket).

Good point - I did a quick override for the color and font weight, though not font size because I think the Figma proportions are a little off (<100px compared to 130px in css), so 12px looks proportionally too small when implemented.

**Review by jaissica — APPROVED** [review]
Tested locally and looks good to me 🚀

**Pull request merged**